client/src/components/ActivityList/ActivityList.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import "./ActivityList.css";
+
+// Class Based React Component
+class ActivityList extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      classList: "ActivityList__activity-list"
+    };
+    this.momActivityList = ["Spa", "Wine Nights", "Brunch with Other Moms"];
+    this.kidsActivityList = [
+      "Snorkeling",
+      "Hiking",
+      "Island Hopping",
+      "Hawaiian Luaus",
+      "Food Markets",
+      "Beaches"
+    ];
+  }
+
+  renderMomActivities = () => {
+    const momActivities = this.momActivityList.map(activity => {
+      return (
+        <div className="ActivityList__activity ActivityList__mom">
+          <span className="ActivityList__activities">{activity}</span>
+        </div>
+      );
+    });
+    return <div className="ActivityList__activity-container">{momActivities}</div>;
+  };
+
+  renderKidsActivities = () => {
+    const kidsActivities = this.kidsActivityList.map(activity => {
+      return (
+        <div className="ActivityList__activity ActivityList__kids">
+          <span className="ActivityList__activities">{activity}</span>
+        </div>
```

Comment:
This should use the `Card` component

Reply:
A Card component for the ActivityList component wasn't created because the component became orphaned. Since the ActivityList cards were unique to that specific page and there weren't any pages of activities to link these activity cards to, we agreed that this would just be it's own styled text list of activities.